src/tools/rest-frontend/public/assets/translations/en.json

```diff
@@ -82,15 +82,16 @@
     "APP.BREADCRUMBS.MAIN.USERS.DETAILS": "User details",
     "APP.BREADCRUMBS.MAIN.USERS.SEARCH": "User search",
     
+    "APP.CONVERSION.ALTTEXT.FORMATCONF": "You can enter additional configuration parameters for the plugin of the selected format here, they will be appended to the existing plugin configuration. If you enter unrelated configuration parameters, they will be ignored.",
```

Comment:
Let us go for configuration settings.

parameters can easily be confused with function parameters.